R/run_baseline_nowcast_pipeline.R

```diff
@@ -0,0 +1,116 @@
+#' Run `baselinenowcast` pipeline
+#'
+#' This function ingests a long form dataframe containing cases by reference
+#'    date and report date, the data we want to nowcast from, and the
+#'    `baselinenowcast` model specifications and returns a probabilistic
+#'    nowcast of the expected total cases by reference date up until the
+#'    specified nowcast date.
+#'
+#' @param long_df Data.frame containing the following columns: `reference_date`,
+#'    `report_date`, and `confirm` indicating the cases on each reference
+#'    and report date.
+#' @param nowcast_date String indicating the date we want to generate a nowcast.
+#' @param max_delay Integer indicating the maximum delay we want to consider in
+#'   the `baselinenowcast` model.
+#' @param n_history_delay Integer indicating the number of reference time
+#'    observations to use to estimate the delay.
+#' @param n_history_uncertainty Integer indicating the number of reporting
+#'    triangles to use to generate the estimate of the dispersion parameters
+#' @param n_draws Integer indicating the number of draws from the observation
+#'    model to use to generate the expected observed nowcasts. Default is
+#'    `1000`.
+#' @param reporting_triangle_to_borrow_delay Matrix to be used to estimate the
+#'    delay if not the same as the one being nowcasted from, `long_df`.
+#' @param reporting_triangle_to_borrow_uncertainty Matrix to be used to
+#'    estimate the dispersion parameters if not the same as the one being
+#'    nowcasted from, `long_df`.
+#'
+#' @autoglobal
+#' @importFrom baselinenowcast add_obs_errors_to_nowcast
```

Comment:
@seabbs Is this the way we want to handle imports from other functions? 

I have tried to set up the autoglobals using `roxyglobals` but thus far it doesn't seem to be filling in the `R/globals.R` file... will play around with it a bit more...

Reply:
do you need to activate it in the description?